### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by justanotherdot**
* Removes TODO 
* Acts like a quickstart guide specifying integration code and example test

### CODE REVIEW

**Comment by justanotherdot** [comment]
Could probably specify the pytest variant of building tests, too, in README integration code sample.

**Comment by justanotherdot** [comment]
There are some pep8 errors but since this is just README documentation, but in order to make traction on this repo I'll be squashing this as well as #1 into master.

File: README.rst [hunk 1]
```diff
@@ -2,15 +2,31 @@
...
+
+Then write a test:
+
+.. code:: yaml
+
+  name: multipart test
+  POST: /somewhere
+  request_headers:
+    content-type: multipart/form-data
+    data:
```
Comment (cdent) [inline README.rst:28]: This is an indent error. `data` shouldn't be under `request_headers`, but instead at the same level.

Comment (justanotherdot) [reply README.rst:28]: Right you are, will fix.

**Pull request merged**